python/ray/dataframe/dataframe.py

```diff
@@ -260,7 +260,7 @@ def __repr__(self):
         # The split here is so that we don't repr pandas row lengths.
         result = self._repr_helper_()
         final_result = repr(result).rsplit("\n\n", maxsplit=1)[0] + \
-            "\n\n[{0} rows x {1} columns]".format(len(self.index),
+            "\n\n[{} rows x {} columns]".format(len(self.index),
```

Comment:
If we do this change (and others) we'll need to fix the indentation of the next line

Reply:
I'm fine leaving this as is then, though I'd prefer that there be a git hook to require all code to be run through auto-formatters. It'd unify things like the use of certain syntax, indent, and quote styles.